lib/views/layout.html

```diff
@@ -41,8 +42,13 @@ <h1>
             <button id="switch">Switch</button>
           </li>
         </ul>
-      </nav>
-   </header>
+        <% if (typeof tosUrl !== 'undefined'  && tosUrl) { %>
```

Comment:
I'm wondering, what's the `&& tosUrl` for, when it's already not `undefined`?

Reply:
Good point!  It should probably be 
`if (typeof tosUrl === 'string' && tosUrl.trim().length > 1)`